guides/docs/introduction/overview.md

```diff
@@ -35,7 +35,8 @@ Phoenix is made up of a number of distinct parts, each with its own purpose and
     - act as a presentation layer
     - define helper functions, available in templates, to decorate data for presentation
  - [Templates](templates.html)
-    - are what they sound like :)
+    - files that will eventually be served in a response
```

Comment:
Could we change this to:

> files containing the contents that will be served in a response

I this helps differentiate between a template and something like Plug.Static